python_tests/test_python_frontend.py

```diff
@@ -954,12 +954,53 @@ def definition(self):
             def schedule(self):
                 self.sched.split(self.t2, 1, 2)
                 self.sched.merge(self.t2, -2)
+                self.sched.reorder(self.t2, {1: 2, 2: 1})
 
         fd = UserDefSched()
         nvf_user_out = fd.execute(inputs)
         nvf_out = fd.execute(inputs, override_user_schedule=True)
         self.assertEqual(nvf_user_out, nvf_out)
 
+    def test_rfactor_user_schedule(self):
+        inputs = [
+            torch.randn(16, 16, device="cuda"),
+        ]
+
+        class UserDefSched(FusionDefinition):
+            def definition(self):
+                self.t0 = fd.define_tensor(
+                    symbolic_sizes=[16, 16],
+                    contiguous=[True, True],
+                    dtype=DataType.Float,
+                )
+                self.t1 = self.ops.sum(self.t0, axis=-1)
```

Comment:
I think we should sum over two dimensions here since the rfactor needs some selected and non-selected reduction dims to factor out.

Reply:
Doesn't split on the reduction dimension give us that already? or is there something more subtle that I'm missing out: 

```
            def schedule(self):
                self.sched.split(self.t1, -1, 4)
                new_tv = self.sched.reduction_factor(self.t1, [1])
```

Reply:
Oh you're right; my bad. The split gives two reduction axes as you said.